Resources/Prototypes/Entities/Objects/Misc/dat_fukken_disk.yml

```diff
@@ -10,7 +10,7 @@
     sprite: Objects/Misc/nukedisk.rsi
     state: icon
   - type: Item
-    size: 12
```

Comment:
So lines 12 to 15 here are your item component. Delete them (everything up to the next component defined with the -)